localdev/dummy.rs

```diff
@@ -0,0 +1,2 @@
+// Dummy file used for local development build
+fn main() {}
```

Comment:
this can probably live in the dockerfile itself. Something like `echo 'fn main() {}' > dummy.rs`

Reply:
True, true. Changing.

Reply:
Note that I removed it from the `Dockerfile` following https://github.com/FedericoPonzi/Horust/pull/41#discussion_r456797726

Reply:
This is now irrelevant since I no longer need a `dummy.rs` file at all, resolving.

Reply:
Resolving